Library/Formula/creduce.rb

```diff
@@ -0,0 +1,93 @@
+class Creduce < Formula
+  homepage "http://embed.cs.utah.edu/creduce/"
+  url "http://embed.cs.utah.edu/creduce/creduce-2.2.1.tar.gz"
+  sha256 "c2fb1b00feafee20b467674679b68a58bc14556c2f373bb9688150f1fc740072"
+
+  head "https://github.com/rgov/creduce.git"
+
+  depends_on "astyle"
+  depends_on "delta"
+  depends_on "llvm" => "with-clang"
```

Comment:
Doesn't look like it's statically linked. It's used in: https://github.com/csmith-project/creduce/blob/creduce-2.2.1/clang_delta/Makefile.am#L22:L27. Do you know how we might provide that information another way?

The results on my machine:
```shell
$ ./Cellar/llvm/3.5.1/bin/llvm-config --cxxflags
-I/usr/local/Cellar/llvm/3.5.1/include  -D_DEBUG -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -O3  -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-common -Woverloaded-virtual -Wcast-qual
$ ./Cellar/llvm/3.5.1/bin/llvm-config --ldflags
-L/usr/local/Cellar/llvm/3.5.1/lib
$ ./Cellar/llvm/3.5.1/bin/llvm-config --includedir
/usr/local/Cellar/llvm/3.5.1/include
$ ./Cellar/llvm/3.5.1/bin/llvm-config --libs
-lLLVMLTO -lLLVMObjCARCOpts -lLLVMLinker -lLLVMipo -lLLVMVectorize -lLLVMBitWriter -lLLVMIRReader -lLLVMAsmParser -lLLVMTableGen -lLLVMDebugInfo -lLLVMOption -lLLVMX86Disassembler -lLLVMX86AsmParser -lLLVMX86CodeGen -lLLVMSelectionDAG -lLLVMAsmPrinter -lLLVMX86Desc -lLLVMX86Info -lLLVMX86AsmPrinter -lLLVMX86Utils -lLLVMJIT -lLLVMLineEditor -lLLVMMCAnalysis -lLLVMMCDisassembler -lLLVMInstrumentation -lLLVMInterpreter -lLLVMCodeGen -lLLVMScalarOpts -lLLVMInstCombine -lLLVMTransformUtils -lLLVMipa -lLLVMAnalysis -lLLVMProfileData -lLLVMMCJIT -lLLVMTarget -lLLVMRuntimeDyld -lLLVMObject -lLLVMMCParser -lLLVMBitReader -lLLVMExecutionEngine -lLLVMMC -lLLVMCore -lLLVMSupport
$ ./Cellar/llvm/3.5.1/bin/llvm-config --system-libs
-lz -lpthread -ledit -lcurses -lm
```